src/extensions/step_handlers/script_handler/src/script_handler.cpp

```diff
@@ -236,7 +238,7 @@ ADUC_Result ScriptHandlerImpl::Download(const tagADUC_WorkflowData* workflowData
     }
 
     // Invoke primary script to download additional files, if required.
-    result = PerformAction("--action-download", workflowData);
+    result = PerformAction("download", workflowData);
```

Comment:
nit: would it be better to pass an enum value, e.g. { Download, Install, ... } rather than a string, in case these values change, or if PerformAction needs to do actions based on the action, e.g. "if action == Download" is better tahn "if (strcmp(action, "Download"))"